sycl/include/CL/sycl/accessor.hpp

```diff
@@ -309,6 +309,14 @@ class image_accessor
 
   constexpr static bool IsImageAccessAnyRead =
       (IsImageAccessReadOnly || AccessMode == access::mode::read_write);
+ 
+  static_assert( 
+       std::is_same<DataT, cl_int4>::value || 
+       std::is_same<DataT, cl_uint4>::value ||
+       std::is_same<DataT, cl_float4>::value ||
+       std::is_same<DataT, cl_half4>::value,
+        "The data type of an image accessor must be only cl_int4, "
+        "cl_uint4, cl_float4 or cl_half4 from SYCL namespace");
```

Comment:
Looks like you have some spare spaces here in front of string.

Reply:
Thanks for suggestion, I applied clang-format here.